snippets/Responsive-Nuemorphism-Card-Design.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Responsive Nuemorphism Card Design
+tags: interactivity,layout
+---
+
+Creates an awesome nuemorphism effect with cool interactivity to the users. 
+
+- Uses `display: flex` property along with `justify-content: center; align-items: center;` to make the snippet responsive.
+- Creates Nuemorphism effect by using nuemorphism colors like `rgba(255, 255, 255, 0.1)` or `rgba(0, 0, 0, 0.6)` and using nuemorphism shadows. 
+- Uses `:hover` pseudo-selector to make the transitions and interactivity.
+
+```html
+<div class="cont flex">
+    <div class="container flex">
+        <div class="card">
+            <div class="box flex">
+                <div class="content">
+                    <h2>01</h2>
```

Comment:
There is far too much stuff in this snippet. A simple paragraph would suffice to showcase the neumorphism technique.